sentry/src/main/java/io/sentry/protocol/SentryTransaction.java

```diff
@@ -115,6 +115,9 @@ public SentryTransaction(
     this.timestamp = timestamp;
     this.spans.addAll(spans);
     this.measurements.putAll(measurements);
+    for (SentrySpan span : spans) {
```

Comment:
Usually yes, but with this constructor you don't pass the Transaction.
This is mostly for a test that calls this constructor, but also in case we use this constructor in other places in the future